### ISSUE TITLE
404 - Incorrect path while calling any method

### ISSUE THREAD

**Issue by PaNaVTEC** [DrewCarlson/CoinGecko-Kotlin#57]
Hi there this used to work, but somehow now is failing with 404. I have the following code:

```
  val coinGecko = CoinGeckoClient.create()
  println(coinGecko.ping().geckoSays)
```

Yields:
```
drewcarlson.coingecko.error.CoinGeckoApiException: CoinGeckoApiError(code=404, message=Incorrect path. Please check https://www.coingecko.com/api/)
	at drewcarlson.coingecko.CoinGeckoClientImpl.ping(CoinGeckoClientImpl.kt:382)
	at drewcarlson.coingecko.CoinGeckoClientImpl$ping$1.invokeSuspend(CoinGeckoClientImpl.kt)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:503)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.ktor.server.netty.EventLoopGroupProxy$Companion.create$lambda-1$lambda-0(NettyApplicationEngine.kt:260)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:831)
```

Tried with version 0.4.0 and 0.4.1-SNAPHOST

**Comment by DrewCarlson** [comment]
I'm unable to reproduce this error, does it still occur for you?  Do you have a VPN or Proxy configured?

If you still experience this issue, could you enable [HTTP Call logging](https://ktor.io/docs/client-logging.html#configure_plugin) and share the request log?

**Comment by DrewCarlson** [comment]
This is resolved in [v0.4.1](https://github.com/DrewCarlson/CoinGecko-Kotlin/releases/tag/v0.4.1) which will be available soon or the `0.4.2-SNAPSHOT` builds are usable now.
